package.json

```diff
@@ -8,7 +8,8 @@
     "out/bin/**/*"
   ],
   "bin": {
-    "generate-openapi": "out/bin/generate-openapi.js"
+    "generate-openapi": "out/bin/generate-openapi.js",
+    "generate-md": "out/bin/generate-md.js"
```

Comment:
I'd rename executable to `generate-docs` or `generate-endpoint-docs` or similar. The reason is: it'll be global to all module dependencies, e.g. `node_modules/.bin` and thus some degree of ambiguity is possible.